cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -131,6 +134,44 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_repository_branch_crud(appliance, request, wait_for_ansible):
+    """
+    Ability to add repo with branch (without SCM credentials).
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/12h
+        tags: ansible_embed
+    """
+    repositories = appliance.collections.ansible_repositories
+    try:
+        playbooks_yaml = cfme_data.ansible_links.playbook_repositories
+        playbook_name = getattr(request, 'param', 'embedded_ansible')
+        repository = repositories.create(
+            name=fauxfactory.gen_alpha(),
+            url=getattr(playbooks_yaml, playbook_name),
```

Comment:
what is intention of getattr here ? I guess it should work w/o it as well

Reply:
Earlier I have hard-coded the Repository URL, then @digitronik suggested to get it from yamls.